syntax = "proto3";

package stan;

service Stan{
 rpc AskQuestion(QuestionRequest) returns (QuestionResponse) {}
}


message QuestionRequest {
    string question = 1;
}

message QuestionResponse {
    string answer = 1;
    bool was_answered = 2;
}
